### PR TITLE
Delete self-study.md

### PR DESCRIPTION
**Pull request by brianamarie**
## Overview
**TL;DR**
This page was added to a private GitHub repository called `services-web` and is now accessible at services.github.com/resources/self-study.

### Questions
@github/services-operations @github/services-training I will add a redirect to minimize any confusion about links already sent out to the path. Do we know of any existing links in canned replies or anywhere else that will need updating? 

### Next Steps
Awareness of @github/services-training and @github/services-operations (please respond with thumbs up if you don't have time to review formally so I know that you know we all know this page has moved. 😊 ) 

### Review
I will merge with awareness from teams mentioned above and a few reviews. 👍

### CODE REVIEW

**Comment by jordanmccullough** [comment]
> Do we know of any existing links in canned replies or anywhere else that will need updating?

@brianamarie, I've checked both our canned reply resources and none refer to the `self-study` page explicitly. 👍
